integration/resource_lacework_policy_test.go

```diff
@@ -63,7 +63,7 @@ func TestPolicyCreate(t *testing.T) {
 	terraformOptions.Vars = map[string]interface{}{
 		"title":       "lql-terraform-policy-updated",
 		"severity":    "Low",
-		"type":        "Summary",
+		"type":        "Compliance",
```

Comment:
"Violation" is the only policy type accepted by the API, so this will get a failure (I hope). My guess is that you can remove this field from the update and you have a working test.

Reply:
👍